src/CMakeLists.txt

```diff
@@ -795,7 +795,7 @@ if (NUPIC_BUILD_PYEXT_MODULES)
     endif()
 
 
-    set(SWIG_MODULE_${MODULE_NAME}_EXTRA_DEPS ${extra_deps})
```

Comment:
cmake didn't seem to complain about `SWIG_MODULE_${MODULE_NAME}_EXTRA_DEPS` as variable name, but it didn't have the desired effect either.